docs/assert/rejects.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: rejects
+description: Test if the provided promise rejects, and optionally compare the rejection value.
+categories:
+  - assert
+---
+
+## `rejects( promise, expected [, message ] )`
+
+Test if the provided promise rejects, and optionally compare the rejection value.
+
+| name               | description                          |
+|--------------------|--------------------------------------|
+| `promise` (thenable) | promise to test for rejection      |
+| `expectedMatcher`  | Rejection value matcher              |
```

Comment:
This name is inconsistent with the function signature given. I'd prefer we use `expectedMatcher` throughout as it makes more sense semantically.